src/t2iapi/combined/combined_requests.proto

```diff
@@ -70,3 +70,14 @@ message SetModeOfOperationAndSetOperatingModeRequest {
                                             // shall be set
     t2iapi.operation.OperatingMode operating_mode = 4; // OperatingMode value to be set for the operation
 }
+
+/*
+  Request to set the activation state of the system context with the specified handle to the requested value and to set
+  the context association of a context state from the context descriptor with the given handle to the requested value.
+*/
+message SetSystemContextActivationStateAndContextAssociationRequest {
```

Comment:
the specification of the interface seems to me correct. I would only adapt the wording to the biceps standard (as suggested in the other remark)

Reply:
changed